client/lib/upgrades/constants.js

```diff
@@ -1,79 +0,0 @@
-/** @format */
-
-export const action = {
```

Comment:
This file was moved to `action-types`. The single exported object was replaced with named exports, and some were pruned.